packages/google_mobile_ads/lib/src/mobile_ads.dart

```diff
@@ -71,6 +71,18 @@ class MobileAds {
     }
   }
 
+  /// For more details about the volume control, visit
+  /// https://developers.google.com/admob/android/global-settings#video_ad_volume_control
```

Comment:
```suggestion
  /// Sets whether the app is muted.
  ///
  /// Default value is false. For more details...
```

See the dart style guide for comments: https://screenshot.googleplex.com/3t94kYsctCvxwkA